sig-etcd/README.md

```diff
@@ -0,0 +1,117 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# etcd Special Interest Group
+
+Make etcd the best production-ready store for building cloud-native distributed systems and managing cloud-native infrastructure via orchestrators like Kubernetes.
```

Comment:
```suggestion
etcd is a production-ready store for building cloud-native distributed systems and managing cloud-native infrastructure via orchestrators like Kubernetes.
```

I'm not explicitly saying etcd isn't the best, but I don't think best is even a goal of Kubernetes itself. etcd is just one way to implement this type of key value store, but it's one that's currently critical path to the upstream implementation of Kubernetes and that's why we want to bring it under the larger Kubernetes project.

Reply:
Done.